### PR TITLE
[KubeRay][Autoscaler] Fix test_autoscaling_config

### LINKED ISSUE #46178: CI test windows://python/ray/tests:kuberay/test_autoscaling_config is consistently_failing
CI test **windows://python/ray/tests:kuberay/test_autoscaling_config** is consistently_failing. Recent failures: 
	- https://buildkite.com/ray-project/postmerge/builds/5048#01903931-0e65-4035-9011-4ff2141e9861
	- https://buildkite.com/ray-project/postmerge/builds/5053#01903b8e-545e-4ad1-8df8-12b0c071ee20

DataCaseName-windows://python/ray/tests:kuberay/test_autoscaling_config-END
Managed by OSS Test Policy

### PR DESCRIPTION
**Pull request by kevin85421**
## Why are these changes needed?

`test_autoscaling_config` is only for KubeRay. KubeRay doesn't support Windows.

## Related issue number

Closes #46178 

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Review by jjyao — APPROVED** [review]

**Comment by kevin85421** [comment]
@can-anyscale do we have a way to run Windows tests before merging into the master branch? Thanks!

**Comment by jjyao** [comment]
Yes, go to premerge and click `windows tests`. I did it for you.

File: python/ray/tests/kuberay/test_autoscaling_config.py
Comment (kevin85421) [inline python/ray/tests/kuberay/test_autoscaling_config.py:184]: These two functions are unused.

File: python/ray/tests/kuberay/test_autoscaling_config.py
Comment (kevin85421) [inline python/ray/tests/kuberay/test_autoscaling_config.py:477]: This file has already imported pytest

**Comment by can-anyscale** [comment]
another way to do this is to tag the bazel rule with the `no_windows`, e.g. tag https://github.com/ray-project/ray/blob/master/python/ray/tests/BUILD#L73

**Comment by kevin85421** [comment]
The serverless Windows tests have already passed.

**Comment by can-anyscale** [comment]
ok, let's ship to unblock the releases \o/

**Comment by can-anyscale** [comment]
another core tests failed though https://buildkite.com/ray-project/premerge/builds/27246#01905040-64d1-469c-9887-66923f8c4b2f/187-1595, is it related?

**Comment by can-anyscale** [comment]
maybe because of this https://github.com/ray-project/ray/commit/4f40e1d1da076ce6750fd9dae035ca10b2b642b8 not sure

**Pull request merged**
